solutions/go/ping-pong/diff/app/server.go.diff

```diff
@@ -0,0 +1,30 @@
+@@ -1,20 +1,28 @@
+ package main
+
+ import (
+ 	"fmt"
+ 	"net"
+ 	"os"
+ )
+
+ func main() {
+ 	l, err := net.Listen("tcp", "0.0.0.0:6379")
+ 	if err != nil {
+ 		fmt.Println("Failed to bind to port 6379")
+ 		os.Exit(1)
+ 	}
+-	_, err = l.Accept()
++
++	conn, err := l.Accept()
+ 	if err != nil {
+ 		fmt.Println("Error accepting connection: ", err.Error())
+ 		os.Exit(1)
+ 	}
++
++	if _, err := conn.Read([]byte{}); err != nil {
```

Comment:
Since we're not parsing the command and doing something with it, why do it at this stage? This can be omitted here and explained later?

Reply:
Nice catch @marcosnils - done, https://github.com/codecrafters-io/build-your-own-redis/pull/33. 

Amended the explanation too, so we only cover this in stage 3 now: 

<img width="1156" alt="Screenshot 2022-07-29 at 12 00 13" src="https://user-images.githubusercontent.com/3893573/181681628-9c1d4ce6-fd74-4bb6-b14d-a953c9a4cd6a.png">